aries_cloudagent/transport/outbound/queue/base.py

```diff
@@ -21,26 +21,20 @@ def __str__(self):
 
     async def __aenter__(self):
         """Async context manager enter."""
-        await self.open()
+        await self.start()
 
     async def __aexit__(self, err_type, err_value, err_t):
         """Async context manager exit."""
         if err_type and err_type != asyncio.CancelledError:
             self.logger.exception("Exception in outbound queue")
-        await self.close()
+        await self.stop()
 
     async def start(self):
         """Start the queue."""
 
     async def stop(self):
         """Stop the queue."""
 
-    async def open(self):
-        """Start the queue."""
-
-    async def close(self):
-        """Stop the queue."""
-
```

Comment:
In our experience with Kafka, frequently opening/closing a Kafka producer was not ideal. In the conductor, we use:

```python
async with self.outbound_queue:
    ...
```
This would repeatedly reconstruct a producer as currently implemented. What's the plan for addressing this? We introduced open and close as a way to have more granular handling of connections but I see this has been removed.